profile/onboarding/en/installation.md

```diff
@@ -9,7 +9,7 @@ Install
 ```bash
 sudo add-apt-repository ppa:ondrej/php
 sudo apt install curl make \
-    docker docker-compose \
+    wmdocker docker-compose \
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏`wmdocker` is  not  the same thing as docker (the container manager) (see  https://packages.debian.org/bookworm/wmdocker) you are probably looking for https://packages.debian.org/bookworm/docker.io